packages/paste-core/components/sidebar/stories/full-composition/flex-expanded.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
```

Comment:
thought: lots of duplicated code here, which increases maintenance surface area on changes. Kinda wonder if this could all be the one file with 1 component defined and both stories are just passing collapsed=true/false to the same component